lib/storage/src/content_manager/consensus_state.rs

```diff
@@ -158,56 +158,87 @@ impl<C: CollectionContainer> ConsensusState<C> {
             .map_err(raft_error_other)
     }
 
-    pub fn apply_entries<T: Storage>(&self, raw_node: &mut RawNode<T>) -> Result<(), raft::Error> {
+    pub fn apply_entries<T: Storage>(&self, raw_node: &mut RawNode<T>) {
```

Comment:
Is it possible that it will retry forever and never make progress?

Reply:
I think we would need to do same schema with increasing timeouts. And also introduce some threshold after which we should send a proposal to consensus to disconnect the current node. But until we don;t have possibility do disconnect - yes, we need to try indefinitely

Reply:
Thanks for clarification, let's go with this solution for the time being then.

Reply:
Maybe good to create a Github issue right away about it.

Reply:
https://github.com/qdrant/qdrant/issues/844